client/lib/site/index.js

```diff
@@ -50,7 +50,7 @@ Emitter( Site.prototype );
 Site.prototype.attributes = function( attributes ) {
 	attributes = attributes || {};
 
-	for ( var prop in attributes ) {
+	for ( let prop in attributes ) {
```

Comment:
I can if you prefer that. I don't like see and get eshint warnings while I'm working in the code. In fact I did three janitorial PRs in parallel just to clean the code to try to keep this one as clear as possible.